src/migrations/1732280885581-create-test-data.ts

```diff
@@ -0,0 +1,173 @@
+        
+        Riichi mahjong is a japanese variation of the old Chinese 4-player game. This is not the solitaire mahjong which many may know, but a complex and exciting game popular in Japan.
+        
+        Our association is open for everyone interested in Riichi mahjong. Our activities include weekly mahjong sessions during semesters as well as tournaments and excursions.', '@aaltodaigaku', '+358 888 888', 'aaltodaigaku@email.com', 22);
+        INSERT INTO public.association
+        (id, "name", logo, description, telegram, phone, email, membership_fee)
+        VALUES(9, 'Otakut', 'https://www.ayy.fi/sites/g/files/flghsv231/files/styles/o_690w_ah_n/public/2024-07/Otakut-Logo-2013%20%281%29.png?itok=WW8Q-zwZ', 'Otakut is an anime club in Espoo, affiliated with Aalto University''s Student Union since May 1996. We support hobbies related to Japanese animated movies, series and comics.
+        
+        Activities
+        - Anime Evenings
+        - Movie Nights and Marathons
+        - Game and Karaoke Nights
+        - Friday Social Events
+        - Outings and Collaborations
+        - Convention Attendance
+        
+        Events are centered in Otaniemi.
+        ', '@otakut', '+358 888 888', 'otakut@email.com', 11);
+        INSERT INTO public.association
+        (id, "name", logo, description, telegram, phone, email, membership_fee)
+        VALUES(10, 'Aalto-Basket', 'https://www.ayy.fi/sites/g/files/flghsv231/files/styles/o_690w_ah_n/public/2019-05/aaltobasket_logo.png?itok=dGaniM1S', 'Aalto-Basket is the basketball team of Aalto University from Espoo.
+        Our club has long traditions of student sports, being founded way back in 1981.
+        During recent years, the club has grown, and nowadays consists of four teams:
+        
+        • Men’s 1st team: Plays in 2nd division, practices twice a week
+        • Men’s 2nd team: Plays in 4th division, practices twice a week
+        • Men’s 3rd team: Plays in 5th division, practices once a week
+        • Women’s team: Playes in 4th division, practices twice a week
+        
+        Each team is looking for new players for the upcoming season (or just to join our practices) – So, feel free to contact us!', '@aaltobasket', '+358 888 888', 'aaltobasket@email.com', 23);
+        
+
+        INSERT INTO public."event"
+        (id, title, description, picture, "date", "location", price, capacity, association_id)
+        VALUES(1, 'Salsa class', 'Discover the rhythm and passion of salsa dancing in a fun and engaging environment! Whether you''re a complete beginner or an experienced dancer, our Salsa Class welcomes all skill levels to sway to the infectious beats of Latin music. Led by a dynamic instructor, you''ll learn foundational moves, partner work, and styling tips that will leave you feeling confident on the dance floor. Come solo or with a partner—either way, you''re guaranteed to leave with new moves and a smile!
```

Comment:
Same as associations. Don't put a value for id, as that will be automatically generated.